src/native.js

```diff
@@ -480,6 +480,11 @@ export function toOrtbNativeRequest(legacyNativeAssets) {
       continue;
     }
 
+    if (key === 'privacyLink') {
+      ortb.privacy = 1;
```

Comment:
If I say `privacyLink: {required: false}` should this still be set to 1?

Reply:
in ORTB the flag signals support and not whether the link is required, so maybe this is OK - but we should set `required: false` in the inverse logic?

Reply:
My takes on this

> If I say privacyLink: {required: false} should this still be set to 1? 
I'd say the presence of the attribute means that you'd like to set privacy to 1. 

> in ORTB the flag signals support and not whether the link is required, so maybe this is OK - but we should set required: false in the inverse logic? 
Agree, will fix